pkgs/applications/misc/mepo/default.nix

```diff
@@ -16,26 +16,53 @@
 , xdotool
 , bemenu
 , withX11 ? false
+
+# transmits user's location without asking first
+, geoclueSupport ? false , geoclue2
+
+# transmits user's location without asking first
+, reportLocationToMozilla ? false
+
+, gpsdSupport ? true , gpsd
```

Comment:
```suggestion
, geoclueSupport ? false, geoclue2

# transmits user's location without asking first
, reportLocationToMozilla ? false

, gpsdSupport ? true, gpsd
```
Shouldn't we assert something here? not sure what tbh

Reply:
> Shouldn't we assert something here?

Why?

Reply:
Would it work to turn all on or off?

Reply:
Correct: all 2^3 combinations of the three boolean flags are valid.